lib/datapacksjob.js

```diff
@@ -2301,12 +2301,38 @@ DataPacksJob.prototype.deployBulkRecords = async function (dataPack, BulkRecords
         BulkRecords = JSON.parse(JSON.stringify(BulkRecords).replace(/%vlocity_namespace%__/g, ''));
     }
 
-    var result = await self.vlocity.datapacksutils.createBulkJob(objectName, 'insert', BulkRecords, randomNum);
+    const BATCH_SIZE_FOR_BULK_JOB = 1000;
```

Comment:
Is there a reason to choose 1000? Could we be smarter about how we pick this number and avoid batching if we don't need to?

Reply:
@mattgoldspink Yes. I was thinking of checking if it's more than 5~6k, then only we can split.
Also we can increase the count to 4k or above.

Also we can have input from user itself, that they can provide to paginate.

This is just POC and open to improvements. Thanks

Reply:
@lmckusick FYI